tools/clang/test/HLSLFileCheck/d3dreflect/rdat_mintarget/sm68_node_barrier.hlsl

```diff
@@ -0,0 +1,52 @@
+// RUN: %dxilver 1.8 | %dxc -T lib_6_8 %s | %D3DReflect %s | %FileCheck %s -check-prefixes=RDAT
+
+// Ensure that barrier is allowed for node shaders.
```

Comment:
I guess it has that side effect, but the checks seem to be, like the other tests to determine that RDAT data correctly sets the version due to node shaders.

Reply:
This one is to test the bug where node wasn't added to the mask of allowed shader stages for this barrier operation.  This test ensures the function is still considered compatible with node shader when the barrier is used.

In fact, this is the fix that's being waited on so that Amar can enable the RDAT validation in the runtime, which currently breaks node shader tests due to this bug.